cmd/kube-controller-manager/app/options/csrsigningcontroller.go

```diff
@@ -70,5 +84,34 @@ func (o *CSRSigningControllerOptions) Validate() []error {
 	}
 
 	errs := []error{}
```

Comment:
is this the right place to ensure:
* general and individual files are not both provided
* `(len(o.ClusterSigningCertFile) == 0) == (len(o.ClusterSigningKeyFile) == 0)`

Reply:
> is this the right place to ensure:
> 
> * general and individual files are not both provided
> * `(len(o.ClusterSigningCertFile) == 0) == (len(o.ClusterSigningKeyFile) == 0)`

happy to also specify here if you like.

Reply:
> is this the right place to ensure:
> 
> * general and individual files are not both provided
> * `(len(o.ClusterSigningCertFile) == 0) == (len(o.ClusterSigningKeyFile) == 0)`

added